lua/autorun/server/sv_daily_restart.lua

```diff
@@ -15,7 +15,7 @@ local ALERT_NOTIFICATION_ADMIN_NAME = "CFC_DailyRestartAlertAdmin"
 local TESTING_BOOLEAN = false
 
 local SOFT_RESTART_STOP_COMMAND = "!stoprestart"
-local MINIMUM_HOURS_BEFORE_RESTART = 3
+local MINIMUM_HOURS_BEFORE_RESTART = 1
```

Comment:
A comment on this variable would be nice for detail, is this hours after a map change etc